modules/junit-jupiter/src/test/java/org/testcontainers/junit/jupiter/MixedLifecycleTests.java

```diff
@@ -0,0 +1,37 @@
+package org.testcontainers.junit.jupiter;
```

Comment:
unfortunately i had to create a new test class, to have the code representing nearly the same logic as the previous documentation